src/job/mod.rs

```diff
@@ -109,7 +114,7 @@ impl JobLocked {
                 last_updated: None,
                 last_tick: None,
                 next_tick: schedule
-                    .upcoming(Utc)
+                    .upcoming(TIME_ZONE)
```

Comment:
> I guess it wouldn't be enough for full local support, because of next_tick of job will be rewritten with Utc value after first tick.

I tried this and it seems to be working well
code: 
```rust
#[test]
fn test_localtime() {
    let s: Schedule = "0 0 12 * * *".try_into().unwrap();

    for (local, utc) in s.upcoming(chrono::Local).zip(s.upcoming(Utc)).take(5) {
        println!("Local time: {} / Utc Time: {}", local, utc);
    }
}
```
output
```
running 1 test
Local time: 2023-09-06 12:00:00 +08:00 / Utc Time: 2023-09-06 12:00:00 UTC
Local time: 2023-09-07 12:00:00 +08:00 / Utc Time: 2023-09-07 12:00:00 UTC
Local time: 2023-09-08 12:00:00 +08:00 / Utc Time: 2023-09-08 12:00:00 UTC
Local time: 2023-09-09 12:00:00 +08:00 / Utc Time: 2023-09-09 12:00:00 UTC
Local time: 2023-09-10 12:00:00 +08:00 / Utc Time: 2023-09-10 12:00:00 UTC
test job::test_localtime ... ok
```
So I didn't find where the next_tick was been rewriteen. Could you explain a little more about that?